plexupdate.sh

```
@@ -85,50 +87,77 @@ URL_LOGIN=https://plex.tv/users/sign_in
 URL_DOWNLOAD=https://plex.tv/downloads?channel=plexpass
 URL_DOWNLOAD_PUBLIC=https://plex.tv/downloads
 
+usage() {
+        echo "Usage: $(basename $0) [-aCfhkopqsuU]"
+        echo "    -a Auto install if download was successful (requires root)"
+        echo "    -C Cron mode. Only output to stdout on an actionable operation"
+        echo "    -d Auto delete after auto install"
+        echo "    -f Force download even if it's the same version or file"
+        echo "       already exists (WILL NOT OVERWRITE)"
+        echo "    -h This help"
+        echo "    -k Reuse last authentication"
+        echo "    -o 32-bit version (default 64 bit)"
+        echo "    -p Public Plex Media Server version"
+        echo "    -q Quiet mode. No stdout/stderr, only exit codes"
```

This is not true since quiet won't suppress errors in the current patch ?